src/components/dialog-full-screen/dialog-full-screen.component.tsx

```diff
@@ -71,6 +71,8 @@ export interface DialogFullScreenProps extends ModalProps {
   onCancel?: (
     ev: React.KeyboardEvent<HTMLElement> | React.MouseEvent<HTMLButtonElement>
   ) => void;
+  /** An optional prop that allows header to expand in size to take up the full width */
+  hasFullWidthHeader?: boolean;
```

Comment:
**question(non-blocking):** Do we need to introduce this as a prop? I thought by default we'd want to have the heading full width?

Reply:
The optional prop has been removed, and the container will be full width by default.